### ISSUE TITLE
OpenNetMeter Release 0.11.0 - Wont Run - unhandled exception - Expected Folder that Doesn't Exist

### ISSUE THREAD

**Issue by war59312** [Ashfaaq18/OpenNetMeter#54]
OpenNetMeter Release 0.11.0 - Wont Run - unhandled exception on running.

Looks like it assumes a folder exists and crashes when it does not. 

So don't crash, create the folder if really needed. Hope not, would be best to remain portable and use the current folder instead.

That is create and save user.config in the folder app is ran from instead of %LocalAppData%.

The crash:

```
Application: OpenNetMeter.exe
CoreCLR Version: 6.0.622.26707
.NET Version: 6.0.6
Description: The process was terminated due to an unhandled exception.
Exception Info: System.ArgumentException: The directory name 'C:\Users\William\AppData\Local\OpenNetMeter' does not exist. (Parameter 'path')
   at System.IO.FileSystemWatcher.CheckPathValidity(String path)
   at System.IO.FileSystemWatcher..ctor(String path, String filter)
   at OpenNetMeter.ViewModels.DataUsageHistoryVM..ctor()
   at OpenNetMeter.ViewModels.MainWindowVM..ctor(MiniWidgetVM mw_DataContext, ConfirmationDialogVM cd_DataContext)
   at OpenNetMeter.Views.MainWindow..ctor()
   at OpenNetMeter.App.Application_Startup(Object sender, StartupEventArgs e)
   at System.Windows.Application.OnStartup(StartupEventArgs e)
   at System.Windows.Application.<.ctor>b__1_0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.Run()
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run()
   at OpenNetMeter.App.Main()
```

```
Faulting application name: OpenNetMeter.exe, version: 0.11.0.0, time stamp: 0x6283e0e1
Faulting module name: KERNELBASE.dll, version: 10.0.19041.1741, time stamp: 0xe9b4a91b
Exception code: 0xe0434352
Fault offset: 0x0000000000034fd9
Faulting process id: 0x5ce0
Faulting application start time: 0x01d88f222d7febb3
Faulting application path: C:\Users\William\Downloads\OpenNetMeter-0.11.0\OpenNetMeter.exe
Faulting module path: C:\WINDOWS\System32\KERNELBASE.dll
Report Id: d80e2255-ca81-42db-b0b1-742cd8449aa5
Faulting package full name: 
Faulting package-relative application ID: 
```

Even the crash is a bit wrong. "C:\Users\William\AppData\Local\OpenNetMeter" does exist but looking at that, looks like version 0.10 uses a subfolder under OpenNetMeter.

**To Reproduce**
1. Run App

**Expected behavior**
Start Cleanly

**App and Windows Version (please complete the following information):**
OpenNetMeter Release 0.11.0
Windows 10 x64 21H2

**Additional context**
OpenNetMeter Release 0.10.0 starts and runs fine for me.

**Comment by Ashfaaq18** [comment]
Hey @war59312 , does this happen every time you try launching the 0.11.0 version or just randomly?

Also regarding the portability of the application, i was considering on moving the user.config and the saved profiles to the application's own folder. But the problem here is, if the user wants to download and use another version. They've also got to manually move the saved profiles from the old version to this new versions folder.

**Comment by war59312** [comment]
OK, looks like just the first time and runs Ok on 2nd try, notice the folder got created.

Perhaps check of the config exists and if doesn't ask user if they wish to import an old config/profile or create a new one.

**Comment by Ashfaaq18** [comment]
fixed in commit 0ff812f325a8d9cff623bda2685bec0b2e6e8198
